util/src/test/scala/stryker4s/extensions/score/MutationScoreCalculatorTest.scala

```diff
@@ -0,0 +1,35 @@
+package stryker4s.extensions.score
+import stryker4s.Stryker4sSuite
+
+class MutationScoreCalculatorTest extends Stryker4sSuite {
+
+  case object Sut extends MutationScoreCalculator
+
+  private val sut = Sut
+
+  describe("Calculating the mutation score") {
+    it("Should give a mutation score of 0.00 if no mutants were found") {
+      val mutationScore = sut.calculateMutationScore(0, 0)
+
+      mutationScore shouldBe 0.00
+    }
+
+    it("Should give a mutation score of 0.00 if 1 mutant is found and non are detected") {
+      val mutationScore = sut.calculateMutationScore(1, 0)
+
+      mutationScore shouldBe 0.00
+    }
+
+    it("Should give a mutation score of 100.00 when all mutations are killed") {
+      val mutationScore = sut.calculateMutationScore(100, 100)
+
+      mutationScore shouldBe 100.00
+    }
+
+    it("Should give a mutation score of 85.71 when 12 of the 14 mutations are killed") {
+      val mutationScore = sut.calculateMutationScore(14, 12)
+
+      mutationScore shouldBe 85.71
+    }
```

Comment:
Can you also add a test for rounding up?

Reply:
What do you mean round up? We always check on 2 decimals.

Reply:
In this example the result is 85.714. I think we should also test if rounding up works correctly (6/9 to 66.67, for example)

Reply:
I've added the test but I don't see much value in it since we are testing internal bigdecimal rounding code with this. Which is not our job :)

Reply:
We're not testing BigDecimal, imo. We're testing if our class rounds up. Whether that uses BigDecimal rounding or something else doesn't matter